rise-google-sheet.html

```diff
@@ -292,17 +292,15 @@
         this._startTimer();
       },
 
-      _handleNoNetwork: function( resp ) {
+      _handleErrorsWithCachedData: function( resp ) {
```

Comment:
@rodrigopavezi Could we rename this `_handleErrors()`, I don't think we need *WithCachedData*